spec/contracts/snippet_contract_spec.rb

```diff
@@ -0,0 +1,33 @@
+# require 'rails_helper'
+
+# RSpec.describe Snippets::Contracts::SnippetContract do
+#   let(:contract) { described_class.new }
+
+#   describe 'Validation' do
+#     context 'with valid input' do
+#       let(:valid_snippet) { build(:valid_snippet) }
```

Comment:
I would suggest that the default state of the `snippet` factory should be valid.  If you want to create invalid data, use a snippet and override in the test.